nsv13/code/game/objects/items/custom_materials.dm

```diff
@@ -1,6 +1,6 @@
 //Metals
 
-GLOBAL_LIST_INIT(duranium_recipes, list (\
+GLOBAL_LIST_INIT(duranium_recipes, list ( \
 	new/datum/stack_recipe("railgun rail", /obj/item/ship_weapon/parts/railgun_rail, 1, time = 20, one_per_turf = FALSE, on_floor = TRUE), \
 	new/datum/stack_recipe("mac barrel", /obj/item/ship_weapon/parts/mac_barrel, 1, time = 20, one_per_turf = FALSE, on_floor = TRUE), \
```

Comment:
Forgot these were still in here.

Reply:
I believe we do still use some of these in construction of deck guns